serialization/tests/TestSerializeTabulatedFunctions.cpp

```diff
@@ -65,6 +68,35 @@ void testContinuous1DFunction() {
         ASSERT_EQUAL(values[j], values2[j]);
 }
 
+void testPeriodicContinuous1DFunction() {
+    // Create a function.
+
+    double min = 0.0, max = 2.0*M_PI;
+    vector<double> values(60);
+    for (int i = 0; i < (int) values.size(); i++)
+        values[i] = sin(2.0*M_PI*i/(values.size()-1));
+
+    Continuous1DFunction function(values, min, max, true);
+
+    // Serialize and then deserialize it.
+
+    stringstream buffer;
+    XmlSerializer::serialize<Continuous1DFunction>(&function, "Function", buffer);
+    Continuous1DFunction* copy = XmlSerializer::deserialize<Continuous1DFunction>(buffer);
+
+    // Compare the two forces to see if they are identical.
+
+    double min2, max2;
+    vector<double> values2;
+    copy->getFunctionParameters(values2, min2, max2);
+    ASSERT_EQUAL(min, min2);
+    ASSERT_EQUAL(max, max2);
+    ASSERT_EQUAL(values.size(), values2.size());
+    for (int j = 0; j < (int) values.size(); j++)
+        ASSERT_EQUAL(values[j], values2[j]);
+
```

Comment:
This should also check that `getPeriodic()` returns the correct value.  Same for the other tests below.

Reply:
Ok.